mkdocs.yml

```diff
@@ -0,0 +1,71 @@
+site_name: k0s Documentation
+site_description: Documentation for k0s, the Zero friction Kubernetes distribution.
+site_author: Mirantis, Inc.
+site_url: https://docs.k0sproject.io
+docs_dir: docs/
+repo_name: GitHub
+repo_url: https://github.com/k0sporoject/k0s
+copyright: Copyright &copy; 2020 <a href="https://mirantis.com/">Mirantis Inc.</a> - All rights reserved.
+edit_uri: ""
+nav:
+  - Overview: README.md
+  - Creating A Cluster: create-cluster.md
```

Comment:
@trawler - this is not valid. A top level item can either refer to a doc or just be a top level item with child items referring to docs.